builtin/providers/azurerm/resource_arm_storage_account_test.go

```diff
@@ -357,3 +383,21 @@ resource "azurerm_storage_account" "testsa" {
         environment = "production"
     }
 }`
+
+var testAccAzureRMStorageAccountNonStandardCasing = `
```

Comment:
One thing I learned from a previous PR to hashicorp is that for new tests, it is recommended to wrap the test config in a function instead of using a package level variable. It is easier to catch missing spots with a func when the parameters change. Search for testAccAzureRMExpressRouteCircuit_basic in latest hashicorp master as an example.